### DISCUSSION TITLE
Load .env file automatically

### DISCUSSION THREAD

**Discussion by heindl-solutions** [chrisant996/clink#391] [Ideas]
I am about to bring myself to putting a `.env` file in the root directory of my new projects.
I think it would be great if project environment variables got loaded automatically whenever a project directory is entered.

Probably clink can be extended to look for` .env `files in the current directory and its parents and load the file automatically.

Links:
https://askubuntu.com/a/1349107
https://stackoverflow.com/a/72001469

**Comment by chrisant996** [comment]
That would be better as a custom Lua script, not built into Clink.
